### ISSUE TITLE
Phase 2 : Ripeness Prediction: Train ripeness classification model

### ISSUE THREAD

**Issue by defender-777** [defender-777/dragon-fruit-ai#10]
Train the EfficientNet model using the prepared dataset to classify dragon fruit into ripeness categories (e.g., Unripe, Semi-Ripe, Ripe).

Use appropriate loss functions (CrossEntropyLoss) and optimizers (Adam), and monitor training using validation metrics.

Expected Output:

Trained classification model
Training logs (loss, accuracy)
